method-discover.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * docs/getting-started-guide/master.adoc
```

Comment:
@HagayVider1 
I don't see this section in the preview. Looks like you haven't included this module in getting-started-guide/master.adoc.

Reply:
@HagayVider1 
Sorry, my mistake. It does appear in the preview. I thought it was a new module.
So what is new here? The heading level?